modules/system/console/OctoberInstall.php

```diff
@@ -130,6 +131,23 @@ protected function setupAdvancedValues()
         $this->writeToConfig('app', ['debug' => $debug]);
     }
 
+    protected function askToInstallPlugins() {
+        if ($this->confirm('Do you wish to install October.Drivers plugin?', false)) {
```

Comment:
Do you mind also changing the wording of this and the other user question to "Install the October.Drivers plugin?"